core/types/src/quote.rs

```diff
@@ -112,6 +112,12 @@ pub struct RawQuote<'a> {
     bytes: &'a [u8],
 }
 
+impl<'a> AsRef<[u8]> for RawQuote<'a> {
```

Comment:
not sure I like the AsRef.  I considered implementing a ptr and length method, but it didn't really seem to provide more over `AsRef`.

Reply:
What's the use case for this impl? I am wondering if maybe what we want is a cheap `Serialize` implementation...